canonical/assets.ubuntu.com
# Correct Horse Battery Staple easter egg

## QA

Visit `localhost:8012/v1/?token=correct%20horse%20battery%20staple`
(Make sure you turn off any JSON prettifying plugins which might mess up the output)
